packages/reef-knot/src/packages/connect-wallet-modal.ts

```diff
@@ -0,0 +1 @@
+export * from '@reef-knot/connect-wallet-modal';
```

Comment:
So we could install just the one package `reef-knot` in the external app. And import the modal like this: `import .. from reef-knot/connect-wallet-modal`. See `exports` field in `package.json`.
Otherwise, we will have warnings like "you need to explicitly include `@reef-knot/connect-wallet-modal` in package.json" and we will get the situation when we have both `reef-knot` and `@reef-knot/connect-wallet-modal` installed at once. This is similar to the `ethers` packages issue.